### DISCUSSION TITLE
Quartz Schema Creation On Startup

### DISCUSSION THREAD

**Discussion by philbondarchuk** [quartznet/quartznet#1311] [Q&A] [answered]
Hello,

How can I create the Quartz Schema on start up if it does not exist without having to run the scripts manually? Is there a way to do this?

Thank you

Phil

**Comment by lahma** [comment, marked as answer]
There's no built-in way. For development environment I would use just a docker-compose setup creating against desired database, for others I'd some sort of migration framework.

**Reply by philbondarchuk** [reply]
OK, thank you!
